src/app/import/components/upload/upload.component.html

```diff
@@ -63,6 +64,7 @@
     <ng-container *ngTemplateOutlet="dataSourcesUpload"></ng-container>
     <ng-container *ngTemplateOutlet="upload"></ng-container>
     <ng-container *ngTemplateOutlet="fileValidation"></ng-container>
+    <ng-container *ngTemplateOutlet="cryptedField"></ng-container>
```

Comment:
since the kern-crypt field is a one liner anyway why is this put into a container?

Reply:
no need for container, I changed this to template